modules/watchRTC/WatchRTC.ts

```diff
@@ -0,0 +1,82 @@
+
+import Logger from '@jitsi/logger';
+import watchRTC from '@testrtc/watchrtc-sdk';
+
+import { isAnalyticsEnabled, isRtcstatsEnabled, isWatchRTCEnabled } from './functions';
+import { IWatchRTCConfiguration } from './interfaces';
+
+const logger = Logger.getLogger(__filename);
+
+/**
+ * Class that controls the watchRTC flow, because it overwrites and proxies global function it should only be
+ * initialized once.
+ */
+class WatchRTCHandler {
+    options?: IWatchRTCConfiguration;
+    /**
+     * Initialize watchRTC, it overwrites GUM and PeerConnection global functions and adds a proxy over them
+     * used to capture stats.
+     *
+     * @param {Object} options - Init options.
+     * @returns {void}
+     */
+    init(options: any): void {
+
+        if (isWatchRTCEnabled(options)) {
+
+            if (!isAnalyticsEnabled(options)) {
+                logger.error('Cannot initialize WatchRTC when analytics or third party requests are disabled.');
+                return;
+            }
+
+            if (isRtcstatsEnabled(options)) {
+                logger.error('Cannot initialize WatchRTC when RTCStats is enabled.');
+                return;
+            }
+
+            // watchRTC "proxies" WebRTC functions such as GUM and RTCPeerConnection by rewriting the global
```

Comment:
This is no longer the case. We used to store `window.PeerConnection` but we no longer do. Or did we miss a spot?

Reply:
I am not sure I understand. Is this about the verbose comment written and behaviour of the lib jitsi meet written within the comment ?!

Reply:
It's about the comment.